proto/chart/v1/stream.proto

```diff
@@ -36,6 +36,13 @@ message ChartsAndDimensionsAck {
   string node_id = 2;
   // the last verified stored message's seq_id
   uint64 last_sequence_id = 3;
+  // the message type that triggered this ack message
+  ChartsAndDimensionsAckSource source = 4;
+}
+
+enum ChartsAndDimensionsAckSource {
+  CHART_INSTANCE_UPDATED = 0;
```

Comment:
There are some guidelines regarding enum, but the tool we are using for validation, but is currently broken :/  https://docs.buf.build/style-guide/#enums

Personally, I am strong about the zero enum which should have a special prefix and not used for anything else.

> All enums should have a zero value. Proto3 does not differentiate between set and unset fields, so if an enum field is not explicitly set, it defaults to the zero value. If an explicit zero value is not part of the enum definition, this will default to the actual zero value of the enum. For example, if you had:

Reply:
I totally really agree with the rationale here and I was also a bit worried on implicitly having the first value as default, but the solution would be to introduce a dummy default enum value (For example `UNSPECIFIED`) and assign it to zero ? 🤔

Of course then the receiving side should always check this field's value and handle it accordingly.